diff --git a/dist/index.mjs b/dist/index.mjs
index 4207adc904ae461d93b2e82d9791cdf957b527e6..3fc3079bd6dc1a6d3b68bb065380cbff905d3f01 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -13,6 +13,10 @@ import { sync as resolve } from "resolve";
 import path from "path";
 import { execaSync } from "@esm2cjs/execa";
 import globalDirs from "global-dirs";
+import { fileURLToPath } from "url";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
 function isPathInside(childPath, parentPath) {
   if (!parentPath)
     return false;
diff --git a/package.json b/package.json
index b240a49d0cbf371b8fcc45820e43eea7ea10686b..54d9970503353e15063740088c6dde6ed5cd45d4 100644
--- a/package.json
+++ b/package.json
@@ -15,6 +15,13 @@
   ],
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+   "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    }
+  },
   "files": [
     "dist"
   ],
